src/etotal.ado

```diff
@@ -227,6 +250,10 @@ program define etotal
 	matlist `results', showcoleq(combined) coleqonly  		///
 					   rspec("`rspec'") cspec("`cspec'")
 	display ""
+	if "`e(vcetype)'" != "" {
+	    display as text "note: `e(vcetype)' variance estimation"
+		display ""
```

Comment:
Is the second `display` intentional for spacing?

Reply:
yep